### PR TITLE
Update CI docs

### PR DESCRIPTION
**Pull request by kthoms**
- MWE 2.10
- maven-clean-plugin 3.1.0
- Version matrix

Signed-off-by: Karsten Thoms <karsten.thoms@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.doc/contents/350_continuous_integration.html [hunk 1]
```diff
@@ -254,6 +254,54 @@ <h2 id="maven-tycho-hints">Maven Tycho Hints</h2>
     </tr>
   </thead>
   <tbody>
+    <tr>
+      <td><a href="http://download.eclipse.org/modeling/tmf/xtext/updates/releases/2.17.0/">2.17.0</a></td>
+      <td><a href="http://download.eclipse.org/releases/2019-03">2.16.0</a> (2.12.0)</td>
+      <td><a href="http://download.eclipse.org/modeling/emft/mwe/updates/releases/2.9.0/">2.9.0</a> (2.7.1)</td>
```
Comment (szarnekow) [inline org.eclipse.xtext.doc/contents/350_continuous_integration.html]: Should this be 2.10?

File: org.eclipse.xtext.doc/contents/350_continuous_integration.html [hunk 2]
```diff
@@ -254,6 +254,54 @@ <h2 id="maven-tycho-hints">Maven Tycho Hints</h2>
...
+      <td><a href="http://download.eclipse.org/modeling/m2t/xpand/updates/releases/R201605260315">2.2.0</a> (1.4)</td>
+      <td><a href="http://download.eclipse.org/releases/2019-03">4.11.0</a> (4.7.3)</td>
```
Comment (szarnekow) [inline org.eclipse.xtext.doc/contents/350_continuous_integration.html]: Why is this the same URL?

Comment (kthoms) [reply org.eclipse.xtext.doc/contents/350_continuous_integration.html]: All the features are on the SimRel repository. No need to use more repos.

File: org.eclipse.xtext.doc/contents/350_continuous_integration.html (hunk 1 above)
Comment (kthoms) [reply org.eclipse.xtext.doc/contents/350_continuous_integration.html]: right, this should be changed.

**Comment by cdietrich** [comment]
@kthoms where is the pr for the source?

**Comment by kthoms** [comment]
will be replaced by separate PR

**Pull request closed without merging**